gcc/config/aarch64/aarch64.h

```diff
@@ -430,20 +439,20 @@ enum class aarch64_feature : unsigned char {
    true but not until function epilogues have been generated.  This ensures
    that X30 is available for use in leaf functions if needed.  */
 
-#define CALL_USED_REGISTERS				\
-  {							\
-    1, 1, 1, 1,   1, 1, 1, 1,	/* R0 - R7 */		\
-    1, 1, 1, 1,   1, 1, 1, 1,	/* R8 - R15 */		\
-    1, 1, 0, 0,   0, 0, 0, 0,	/* R16 - R23 */		\
-    0, 0, 0, 0,   0, 1, 1, 1,	/* R24 - R30, SP */	\
-    1, 1, 1, 1,   1, 1, 1, 1,	/* V0 - V7 */		\
-    0, 0, 0, 0,   0, 0, 0, 0,	/* V8 - V15 */		\
-    1, 1, 1, 1,   1, 1, 1, 1,   /* V16 - V23 */         \
-    1, 1, 1, 1,   1, 1, 1, 1,   /* V24 - V31 */         \
-    1, 1, 1, 1,			/* SFP, AP, CC, VG */	\
-    1, 1, 1, 1,   1, 1, 1, 1,	/* P0 - P7 */		\
-    1, 1, 1, 1,   1, 1, 1, 1,	/* P8 - P15 */		\
-    1, 1			/* FFR and FFRT */	\
+#define CALL_USED_REGISTERS					\
+  {								\
```

Comment:
please fix formatting

Reply:
> please fix formatting

You wll need to give me a clue on what formatting needs fixing? :)

Reply:
you can see it here
https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/pull/12/files

Reply:
I mean could you explain what formatting is incorrect?

Reply:
There is a problem with white spacing 

![image](https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/assets/1791423/443bc492-6c1f-483c-ba06-8c659b03cc26)

Reply:
Sorry - I am still not sure what you mean. There is an extra tab there to keep '\' characters aliged after the marco was modified. My experience is the GNU folks like to keep macros aligned in that way. That is an intentional change obviously. 

Is it that you dont want dont want '\' characters aliged? Or something else?

Reply:
It might be that whitespace is hidden in your settings

![image](https://github.com/Windows-on-ARM-Experiments/gcc-woarm64/assets/1791423/2bb2219e-00f1-4c77-bb9b-85b3fed1a3a9)

Reply:
Sorry. I still dont understand what is the whitespace issue.
Could you say the line of code and exactly what is the issue?